magento/magento2
# Fix for #26284 - Incorrect Tax showing for online Credit Memo

### Description (*)
Fix for Issue: Incorrect Tax showing for online Credit Memo #26284
Link: Fixes magento/magento2#26284

The issue was caused by the fact that the $allowedTax and $allowedBaseTax variables were set to the same values without taking into consideration if the current creditmemo is being created for an invoice. The commit makes sure to subtract the refunded tax for any credit memo that is associated with the current invoice.


### Manual testing scenarios (*)

1. Place an order with at least 2 products.
2. Create 2 different invoices, each covering a single product.
3. Create a credit memo for one of the invoice.

**Result:** The credit memo tax total should only reflect the non-refunded invoice tax total instead of the tax for the entire order.

### Questions or comments
Ideally a getCreditmemosCollection(), getTaxRefunded() and getBaseTaxRefunded() functions could also be integrated directly into the Magento\Sales\Model\Order\Invoice class based on the submitted changes.

### Contribution checklist (*)
